website/members/locale/nl/LC_MESSAGES/django.po

```diff
@@ -680,6 +711,95 @@ msgstr ""
 "verlengd hebt, maar dat dit nog niet is verwerkt: in dat geval hoef je niets "
 "te doen."
 
+#: templates/members/email/information_check.html
+msgid "membership information check"
+msgstr "controle gegevens lidmaatschap"
+
+#: templates/members/email/information_check.html
+msgid ""
+"\n"
+"                    We currently have the following information about you in "
+"our database.<br>\n"
+"                    Via this email we'd like to ask you to check the "
+"accuracy of this data.<br>\n"
+"                    "
+msgstr ""
+"\n"
+"                    Hieronder vind je de (meeste) gegevens die we op dit "
```

Comment:
The translation don't fully match de original

Reply:
I believe it does? https://github.com/svthalia/concrexit/blob/4ea3c0eda1ad0ca9476639fbd65dfb8eee26c286/website/members/locale/nl/LC_MESSAGES/django.po#L840